project/Templates/informative-document.rst

```diff
@@ -0,0 +1,64 @@
+.. This is a template for an informative/general use document. 
+
+.. Review the README in this document's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Name-of-Primary-Author*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *List-of-contributors*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Templates-Informative-Document:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+####################
+Informative Document
+####################
+
+.. note::
+    This is a document template file that is associated with a template directory structure. This note should be deleted when the section is properly populated.
+
+.. _Informative-Document-Overview:
+
+Overview
+^^^^^^^^
+
+.. This section should provide a brief, top-level description of the document's purpose and utilization. 
+
+.. _Informative-Document-Main-Information-Section:
+
+Main Infomation Section
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+   Include utilization of admonishments (caution, warning, etc.)
+
+Present the information for this document. 
+Include links or further resources for additional follow-up.  
+
+.. warning::
+    Keep this in mind when referring to this information. 
+
+.. _Informative-Document-Main-Information-Section-Subsection:
+
+Subsection
+==========
+
```

Comment:
Maybe add here two Subsection 1 and Subsection2 to show the general structure?

Reply:
👍